frontend/src/app/related-items/related-items.component.html

```diff
@@ -0,0 +1,24 @@
+<mat-card *ngIf="!loading && (relatedResources.length > 0 || relatedStudies.length > 0)">
+  <mat-card-title>Related</mat-card-title>
+  <mat-card-content>
+    <mat-list>
+
+      <ng-container *ngIf="relatedResources.length > 0">
+        <h3 mat-subheader>Resources</h3>
+        <mat-list-item>
+          <a  mat-line [routerLink]="['/resource', relatedResources[0].id]">{{relatedResources[0].title}}</a>
+          <p mat-line>{{relatedResources[0].description}}</p>
```

Comment:
Rather than showing the title and full description of only one resource and one study, I think it would be nice to show abbreviated versions of several resources and several studies (Title, and one line of the description, maybe?). We've talked about this feature being one of exploration and discovery, so if we show more potentially related items, we're more likely to show the user something of interest to them.

Reply:
This could be something that we revisit with issue #236 rather than delaying this PR for it.